packages/fixie/src/main.ts

```diff
@@ -179,14 +179,36 @@ sources
   .description('Add a source to a corpus.')
   .option('--max-documents <number>', 'Maximum number of documents to crawl')
   .option('--max-depth <number>', 'Maximum depth to crawl')
+  .option('--include-patterns <pattern...>', 'URL patterns to include in the crawl')
+  .option('--exclude-patterns <pattern...>', 'URL patterns to exclude from the crawl')
```

Comment:
OOC does this work if you have multiple include patterns and also multiple exclude patterns?

Reply:
Yes